server/tserver/src/main/java/org/apache/accumulo/tserver/scan/NextBatchTask.java

```diff
@@ -89,14 +88,16 @@ public void run() {
       }
     } catch (TooManyFilesException | SampleNotPresentException e) {
       addResult(e);
-    } catch (OutOfMemoryError ome) {
-      Halt.halt("Ran out of memory scanning " + scanSession.extent + " for " + scanSession.client,
-          1);
-      addResult(ome);
-    } catch (Throwable e) {
+    } catch (Exception e) {
```

Comment:
Should this be more specific, like RuntimeException?

Reply:
I can do that.